src/sidebar.ts

```diff
@@ -97,149 +106,151 @@ const quickstartSidebar = {
       collapsible: false,
       link: {
         type: "doc",
-        id: "getting-started/ory-network-oauth2",
+        id: "guides/oauth2-openid-connect",
```

Comment:
Changed this because `getting-started/ory-network-oauth2` has a custom sidebar label, so now we added an autogenerated page